config.yaml

```diff
@@ -15,9 +15,14 @@ repositories:
   - name: mlkem-native
     teams:
       bots: admin
-      pqcp-native-admin: admin
-      pqcp-native-contributors: write
-      pqcp-native-maintainers: maintain
```

Comment:
Why was this removed?

Reply:
I renamed it from pqcp-native-XXX to pqcp-mlkem-native-XXX
(as written in the PR)

Reply:
No, this one was lost

Reply:
Ahh maintainers and admins were the same set of people. I didn't see a reason to keep both.

Reply:
Ok, thanks! Just wanted to make sure we didn't accidentally drop a line here.